BlogTemplate/Pages/Post.cshtml

```diff
@@ -90,7 +90,8 @@
     }
 
     <hr class="commentsDivider" />
-    @if (Model.Post.IsPublic)
+    @if (Model.IsCommentsFull){}

```

Comment:
Would it make more sense to check if !Model.IsCommentsFull and Model.Post.IsPublic, rather than checking if Model.IsCommentsFull and doing nothing in that case?